### PR TITLE
Always return to the templates current tab selection

### PR DESCRIPTION
**Pull request by briri**
Make sure the current tab selection is retained when editing a template. #1042

### CODE REVIEW

**Comment by jollopre** [comment]
@briri there are conflicts to be resolved

**Comment by briri** [comment]
Rebased against dev to resolve conflict

File: app/controllers/phases_controller.rb [hunk 1]
```diff
@@ -24,7 +24,8 @@ def edit
     else
       render('/phases/edit', locals: { plan: plan, phase: phase, readonly: readonly, 
                                        question_guidance: plan.guidance_by_question_as_hash,
-                                       guidance_groups: guidance_groups })
+                                       guidance_groups: guidance_groups,
+                                       current_tab: params[:r] || 'all-templates' })
```
Comment (jollopre) [inline app/controllers/phases_controller.rb]: this action is used for displaying a plan's phase, i.e. the answering of a plan. Please, remove changes applied here.

**Comment by briri** [comment]
removed the param from the plans/phase controller action

**Pull request merged**